akka-http2-support/src/main/scala/akka/http/impl/engine/http2/framing/Http2FrameParsing.scala

```diff
@@ -100,7 +100,7 @@ private[http2] class Http2FrameParsing(shouldReadPreface: Boolean) extends ByteS
               if (payload.hasRemaining)
                 throw new Http2Compliance.IllegalPayloadInSettingsAckFrame(payload.remainingSize, s"SETTINGS ACK frame MUST NOT contain payload (spec 6.5)!")
 
-              SettingsAckFrame
+              SettingsAckFrame(Nil)
```

Comment:
That's the point, where we would have needed to keep track of the outgoing settings to put them in here again. Maybe add that in a comment if we decide to allow dynamic setting updates later on.